Symfony/CS/Fixer.php

```diff
@@ -311,6 +312,48 @@ public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileC
         return $fixInfo;
     }
 
+    /**
+     * @param \SplFileInfo     $file
+     * @param Tokens           $tokens
+     * @param FixerInterface[] $fixers
+     *
+     * @return string[]
+     */
+    public function fixFileTokens(\SplFileInfo $file, Tokens $tokens, array $fixers)
```

Comment:
Well, just create a file in a temporary folder for the integration test, which will allow you to run the whole logic.

AbstractFixerTestBase uses only the FixerInterface implementation because it is meant to write unit tests for the FixerInterface implementations, and so should run these classes only, not the whole library